packages/shared/src/BootcAPI.ts

```diff
@@ -29,4 +29,5 @@ export abstract class BootcApi {
   abstract openFolder(folder: string): Promise<boolean>;
   abstract generateUniqueBuildID(name: string): Promise<string>;
   abstract openLink(link: string): Promise<void>;
+  abstract logUsage(eventName: string, data?: Record<string, unknown> | undefined): Promise<void>;
```

Comment:
to differentiate it from the other commands, we should name it telemetryLogUsage https://github.com/containers/podman-desktop-extension-ai-lab/blob/main/packages/shared/src/StudioAPI.ts#L57C12-L57C29

Reply:
Renamed. As per other comment I added telemetryLogError as well, I just hadn't added that since we aren't using it yet.

Ready for another review.